### ISSUE TITLE
Experimental keys on scalar is now forbidden

### ISSUE THREAD

**Issue by eserte** [dimentox/Log-Log4perl-Appender-Graylog#1]
The test suite fails for perl 5.24.0 and newer:
```
#   Failed test 'use Log::Log4perl::Appender::Graylog;'
#   at t/00_compile.t line 4.
#     Tried to use 'Log::Log4perl::Appender::Graylog'.
#     Error:  Experimental keys on scalar is now forbidden at /home/cpansand/.cpan/build/2017061100/Log-Log4perl-Appender-Graylog-1.3-SW_KtT/blib/lib/Data/DTO/GELF.pm line 74.
# Compilation failed in require at /home/cpansand/.cpan/build/2017061100/Log-Log4perl-Appender-Graylog-1.3-SW_KtT/blib/lib/Log/Log4perl/Appender/Graylog.pm line 16.
# BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2017061100/Log-Log4perl-Appender-Graylog-1.3-SW_KtT/blib/lib/Log/Log4perl/Appender/Graylog.pm line 16.
# Compilation failed in require at t/00_compile.t line 4.
# BEGIN failed--compilation aborted at t/00_compile.t line 4.
# Looks like you failed 1 test of 1.
t/00_compile.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
...
```

**Comment by eserte** [comment]
I still see the error with 1.4

**Comment by dimentox** [comment]
Will look into it ty

**Comment by dimentox** [comment]
Fixed in v1.7 TYVM!
